### DISCUSSION TITLE
Event on ScrollArea end reached

### DISCUSSION THREAD

**Discussion by Sajarin-M** [mantinedev/mantine#3088] [Q&A] [answered]
what is the best way to trigger an event on scroll area end reached ?

**Comment by rtivital** [comment, marked as answer]
You can subscribe to scroll position changes and measure viewport height – https://mantine.dev/core/scroll-area/#subscribe-to-scroll-position-changes

**Reply by Sajarin-M** [reply]
how can i get the scroll target height ?

**Reply by rtivital** [reply]
Something like `viewportRef.current.clientHeight`
